portal-web/test/functional/com/liferay/portalweb/macros/FormsAdmin.macro

```diff
@@ -62,6 +62,70 @@ definition {
 			outputParameterType2 = "${outputParameterType2}");
 	}
 
+	macro editDataProviderLocalNetwork {
```

Comment:
@Jamersonze Why didn’t you use the existing `editDataProvider` macro?

Reply:
The existing macro edits non required fields when its not necessary like username and password. Besides, i wanted to upgrade the existing macro but this would render to break other testcases already using it. So, i created that macro. that can now deal with all the fields above inputs and outputs.